src/procedures/__tests__/CreateErc20DividendDistribution.ts

```diff
@@ -32,7 +37,12 @@ describe('CreateErc20DividendDistribution', () => {
   let tokenFactoryMockStub: SinonStub<any, any>;
```

Comment:
This stub doesn't seem to be used anywhere, so I don't see the need to assign it to a variable